.github/workflows/redis-stack-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Redis_stack_tests
```

Comment:
This file isn't needed anymore.

Reply:
So we're changing original workflow to run tests over `redis-stack` instead of `redis-server`?

Reply:
Keep `tests.yml` and call this `stack.yml`.

Reply:
`Redis stack` only available since `Redis` server version >= 6.x, so it means we can only run tests against v6.x, 7.x